curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff83.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 683ec95f2f8781355556ff83
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+You can see that the first call returns `[3]`. That's because `3` is the middle value in `[1, 2, 3, 4, 5]`.
+
+On the other hand, the second call returns `This shows because the value is farther than the middle value to the right or left`. This happens because, after checking the middle value, the loop currently lacks the logic to narrow the search range by updating the `low` or `high` variables. It only acts as a single midpoint check and then finishes without further searching.
```

Comment:
```suggestion
On the other hand, the second call returns an empty list. This happens because, after checking the middle value, the loop currently lacks the logic to narrow the search range by updating the `low` or `high` variables. It only acts as a single midpoint check and then finishes without further searching.
```